README.md

```diff
@@ -27,9 +27,8 @@ It was heavily inspired by the ftrace framework of the Linux kernel and
 the name uftrace stems from the combination of user and ftrace.
 
 It can record data from:
-- User space functions by dynamically patching functions at runtime, or with code
-  compiled with with `-pg`, `-fpatchable-function-entry`, `-mfentry`, `-mnop-mcount`,
-  `-finstrument-functions`, or using clang with `-fxray-instrument`.
+- Functions in the executable by dynamically patching functions at runtime using `-P.`.
```

Comment:
uftrace can trace functions inside share libraries as well so I think the "executable" is not enough. I think we better go back to "User space functions" instead rather than "Functions in the executable and shared libraries".